src/sparseml/transformers/finetune/text_generation.py

```diff
@@ -148,11 +148,13 @@ def intialize_model_from_path(
         cache_dir=model_args.cache_dir,
         revision=model_args.model_revision,
         use_auth_token=True if model_args.use_auth_token else None,
+        tie_word_embeddings=False,
```

Comment:
per @Satrat let's add a warning that this is disabled and can be turned on if using the new modifier

Reply:
Is this argument always `False`? Wouldn't it be nice to somehow allow users to override this? or am I missing something here? 
On a side note it would be nice to have a short comment explaining why we need this to be `False` the PR description covers this but I'd like to keep the reason somewhat close to the code? thoughts?